_projects/SP19-Code23

```diff
@@ -0,0 +1,9 @@
+---
+layout: project
+title: Code23
+dl: https://drive.google.com/open?id=1EUgeJBJ1EY9yY6oJ2pVcpUA1cZq7SAc3
```

Comment:
Make sure this includes the Data folder. See the submission instructions [here](https://cs498vr.cs.illinois.edu/assignments/mp1.html#submission-instructions).